build/gtsrb/build.py

```diff
@@ -63,16 +81,32 @@ def custom_step_add_preproc(model, cfg):
     model.graph.node[1].input[0] = new_in_name
     # set input dtype to uint8
     model.set_tensor_datatype(in_name, DataType["UINT8"])
+
+    return model
+
+
+# Insert TopK node to get predicted Top-1 class
+def step_preprocess(model, cfg):
```

Comment:
Please rename to `custom_step_add_postproc` and insert the step like `custom_step_add_preproc` is inserted.

Reply:
Resolved in [c3023fb](https://github.com/Xilinx/finn-examples/pull/109/commits/c3023fbd27e2710e9d3c4367d92c75653a04ee30)